src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -324,17 +325,17 @@ public virtual object Clone()
         // V1 API.
         public static Delegate CreateDelegate(Type type, object target, string method)
         {
-            return CreateDelegate(type, target, method, false, true);
+            return CreateDelegate(type, target, method, false, throwOnBindFailure:true)!; // Cannot return null because it would have thrown
```

Comment:
```suggestion
            return CreateDelegate(type, target, method, false, throwOnBindFailure: true)!; // Cannot return null because it would have thrown
```

Reply:
Nit: The usual formatting is to have a space between `:` and named argument value. Multiple places.